### ISSUE TITLE
Pre-trained batch norm

### ISSUE THREAD

**Issue by drabramow** [bowang-lab/scGPT#22]
When loading in the pre-trained checkpoint I'm finding that no batch normalization parameters are included. 

`Missing key(s) in state_dict: "bn.weight", "bn.bias", "bn.running_mean", "bn.running_var"`

After looking through the pre-trained checkpoint state_dict, these layers are in fact missing, and not even included another name. Is it just a matter of training the weights from scratch when fine-tuning? Thanks.

**Comment by kawairine** [comment]
> Yes, those batchnorm parameters are supposed to be trained from scratch when fine-tuning.

Hi, May I ask what is the rationale behind this? That they are not used in pretraining? 

Many thanks!
